spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/store/redis/RedisTokenStore.java

```diff
@@ -177,15 +177,10 @@ public void storeAccessToken(OAuth2AccessToken token, OAuth2Authentication authe
 				conn.set(refreshToAccessKey, auth);
 				byte[] accessToRefreshKey = serializeKey(ACCESS_TO_REFRESH + token.getValue());
 				conn.set(accessToRefreshKey, refresh);
-				if (refreshToken instanceof ExpiringOAuth2RefreshToken) {
```

Comment:
It's not clear to me if this is an issue. Can you please provide a test demonstrating the issue.